style.css

```diff
@@ -0,0 +1,57 @@
+.container {
+  display: flex;
+  border: 10px;
+  flex-wrap: wrap;
+}
+
+.one {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  color: #6070ff;
+}
+
+.menu-icon {
+  display: flex;
+}
+
+.title {
+  color: #172b4d;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
+
+.paragraph {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  display: flex;
+  align-items: stretch;
+}
+
+.line {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  width: 117px;
+  height: 24px;
+  margin-left: 24px;
+  margin-right: 234px;
+  margin-top: 526px;
+  margin-bottom: 168px;
+}
+
+.social-media-icons {
+  background: #505f79;
```

Comment:
- [ ] Kindly remove this background-color, as it is blocking the icons from displaying